packages/datasource-jsonld/lib/datasources/JsonLdDatasource.js

```diff
@@ -10,6 +10,11 @@ let ACCEPT = 'application/ld+json;q=1.0,application/json;q=0.7';
 class JsonLdDatasource extends MemoryDatasource {
   constructor(options) {
     super(options);
+    if (options.file) {
+      if (!options.file.startsWith('file://'))
```

Comment:
I would suggest to only do this when no `http:` prefix is available as well. Otherwise, publishers that currently use `file` to refer to an external URL will see failures.

Wondering if we should also abstract this into `MemoryDatasource`, so that other datasources (such as n3 and rdfa) can also make use of it.

Reply:
Done!